.github/workflows/sitecheck.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: SiteCheck
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Are you planning to enable on 'next' branch as well?

Reply:
For now, just this branch, since we expect the `next` branch to have different build steps.

Reply:
Most of the doc changes going forward will be in `next`, so that's where checking for broken links is very useful.